cli/src/main/java/org/aya/cli/repl/jline/completer/AyaCompleters.java

```diff
@@ -49,14 +51,23 @@ public Context(@NotNull Repl repl) {
       this.repl = repl;
     }
 
+    private @NotNull Tuple2<String, Boolean> fixWord(@NotNull String word, ParsedLine line) {
+      if (word.startsWith(":") || word.startsWith(Constants.SCOPE_SEPARATOR)) {
```

Comment:
COMMAND_PREFIX

Reply:
This `:` is not a command prefix but a part of scope separator 🤣